pkg/alertmanager/multitenant.go

```diff
@@ -333,7 +333,7 @@ func (am *MultitenantAlertmanager) Stop() {
 // Load the full set of configurations from the server, retrying with backoff
 // until we can get them.
 func (am *MultitenantAlertmanager) loadAllConfigs() map[string]configs.View {
-	backoff := util.NewBackoff(backoffConfig, nil)
+	backoff := util.NewBackoff(nil, backoffConfig)
```

Comment:
Should this be `Background()` instead?

Reply:
+1 and also in scheduler.go.  Always assuming the context is non-nil would also simplify the implementation in backoff.go.